README.md

```diff
@@ -35,11 +35,10 @@ All new backend code and React components should be well tested.
 
 ## Prerequisites
 
-- Node 14
+- Node.js 14 (we require 14 because we want the full internationalization built-in)
 - Yarn
-- PostgreSQL 10  or 12
+- PostgreSQL 12 (10 probably works)
 - Ruby 2.7
-- Ubuntu 18.04, 20.04, 22.04 or equivalent
```

Comment:
I'm a little hesitant to remove this entirely. The point is we're not supporting non-Ubuntu/Debian versions. Maybe a different phrasing can be used though.

Reply:
No problem - Ubuntu reinstated it latest commit.